docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -25,9 +25,6 @@ properties in the file:
 .. code-block:: text
 
     connector.name=oracle
-    // The exact format of connection-url varies by Oracle version. Refer to
-    // the Oracle Database documentation for version-specific information on the
-    // JDBC Thin driver.
```

Comment:
We should keep this comment, because we've seen users copy-paste this configuration and expect it to work out-of-the-box like other connectors, when in reality the Oracle version and configuration may require a very different connection string. The paragraph after this snippet isn't sufficient to call out this issue.

We should just update the `//` into the syntax-appropriate `#`

Reply:
That would be contrary to what we do in all other connectors. No example can ever be useful for all use cases ..